scripts/ci/upload-build

```diff
@@ -2,35 +2,34 @@
 set -ex
 
 # configuration
+AWS_BUCKET='downloads.mesosphere.io'
+BUCKET_PATH='dcos-ui'
 BUILD_VARIANT=${BUILD_VARIANT:-''}
 BRANCH_NAME=${BRANCH_NAME:-'master'}
-DIST_PATH=${DIST_PATH:-'dist'}
-PACKAGE_PATH=${PACKAGE_PATH:-'.'}
-SHA_SLUG=${SHA_SLUG:-$(git rev-parse --short HEAD)}
+PKG_VERSION=$(git tag -l --contains HEAD)
 
-# aws settings
-AWS_BUCKET='downloads.mesosphere.io'
-BUCKET_PATH='dcos-ui'
+if [[ $PKG_VERSION ]]
```

Comment:
what was the reason for changing this script, was there anything which wasnt possible with the old one?

Reply:
on this very variable: we want to have the correct version number represented in out package.json, when pushing a tag, we should also change it there. that said, this change shouldnt be needed.

Reply:
on this very variable: we want to have the correct version number represented in out package.json, when pushing a tag, we should also change it there. that said, this change shouldnt be needed. 🤔

Reply:
This variable will be the tag name. This `if` block just prepends it with the dash. 

We seem to *never* update the package.json version 🙃. That's why we totally removed this from this script. The `tag` will always be here, because *tags* drive our release process.

Reply:
hm, true. but i'd love to see us finally updating the version and using it correctly :(